### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards- Home]: Talkback does not announce the state of 'choose sample' expand/collapse button.

### ISSUE THREAD

**Issue by Bhaskar301101** [microsoft/AdaptiveCards#8419]
### Target Platforms

Android

### SDK Version

2023.01.20.1

### Application Name

AdaptiveCards

### Problem Description

OS: Android 13
Device: S21+ 5G
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.01.20.1

Repro Steps:

1. Turn on Talkback.
2. Open Adaptive Cards Visualizer Application.
3. Home Page will opened.
4. Now Navigate to 'Sample card' button and select it.
5. Now navigate to "choose sample" expands/collapse control and observe the issue.

**Actual Result:**
Talkback does not announce the state choose sample expand/collapse button. Talkback announce like "drop down menu choose sample".

**Expected Result:**
Talkback should announce the state choose sample expand/collapse button. Talkback should announce like "drop down menu ,choose sample, collapsed".

**User Experience:**
Screen reader users will be impacted as they will be unable to understand whether the controls in expanded state or the collapsed state and When the state, value or property of a component is not defined, assistive technology users are often blocked from using the control or knowing what to expect when interacting with the component.

WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/89897257/228476262-7167c0cf-fd02-463f-a0f0-d7d5897b74cf.mp4

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
we don't fix a11y issues on sample app since we don't ship it to the public

**Comment by vagpt** [comment]
As per the above comment closing this issue.
